test/ConfigFactory/getConfig.test.ts

```diff
@@ -18,17 +18,33 @@
 import sinon from 'sinon';
 
 import MosaicConfig from '@openst/mosaic-chains/lib/src/Config/MosaicConfig';
+import GatewayConfig from '@openst/mosaic-chains/lib/src/Config/GatewayConfig';
 import { Config, FacilitatorConfig } from '../../src/Config/Config';
 import ConfigFactory from '../../src/Config/ConfigFactory';
 import assert from '../test_utils/assert';
 import SpyAssert from '../test_utils/SpyAssert';
 import GatewayAddresses from '../../src/Config/GatewayAddresses';
 
-describe('FacilitatorOptionParser.getConfig()', () => {
+describe('ConfigFactory.getConfig()', () => {
   const originChain = '2';
   const auxChain = 3;
   const facilitatorConfigPath = './facilitator-config.json';
   const mosaicConfigPath = './test/mosaic-config.json';
+  const gatewayConfigPath = './gateway-config.json';
+  const mosaicJson = `{"originChain":{"chain":"${originChain}"},"auxiliaryChains":{"${auxChain}":{"chainId": ${auxChain}}}}`;
```

Comment:
This is unreadable, extract this json out in a method.

Reply:
Instead of json, method can return JS object directly because  you are parsing json.

Reply:
Done